BitwardenShared/UI/Vault/Vault/VaultList/VaultListItem.swift

```diff
@@ -103,7 +103,7 @@ extension VaultListItem {
             case .collection:
                 Asset.Images.collections24
             case .folder,
-                 .noFolder:
+                .noFolder:
```

Comment:
⛏️ There shouldn't be any changes in here, restore the space removed so it's aligned to `.folder`.